docs/debug-kernel.md

```diff
@@ -44,3 +44,12 @@ to enable kernel boot logs using the configuration file.
 
    This will result in the `cc-proxy` log being printed on the terminal. This method
    can also be used to capture the logs in a specific `cc-proxy` file.
+
+5. Connect to the Virtual machine console
+
+   If the kernel boots successfully then the virtual machine console can be accessed
```

Comment:
We should avoid using an "if" clause unless we provide options for when the kernel does not boot successfully. 
Let us ensure that our instructions are accurate and assume the reader has followed the instructions exactly:
After the kernel boots, the virtual machine console can be accessed for further debugging.